fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/auto/DruidNavigator.java

```diff
@@ -54,14 +53,13 @@ public DruidNavigator(DruidWebService druidWebService) {
     @Override
     public List<? extends DataSourceConfiguration> get() {
         if (tableConfigurations.isEmpty()) {
-            loadAllDatasources();
+            Future<Response> responseFuture = loadAllDatasources();
             LOG.info("Loading all datasources");
             try {
-                //TODO Druid should ideally return a future, but this accomplishes the same result.
-                countDownLatch.await(5, TimeUnit.SECONDS);
-            } catch (InterruptedException e) {
+                responseFuture.get();
```

Comment:
Rather than 👍 comments, feel free to use the "reaction" capability on comments to show your reactions to things 😉 It generates a little less email spam.